Libraries/WebSocket/RCTWebSocketModule.m

```diff
@@ -85,6 +86,27 @@ - (void)dealloc
   [_sockets[socketID] send:message];
 }
 
+RCT_EXPORT_METHOD(sendBlob:(NSDictionary *)blob socketID:(nonnull NSNumber *)socketID)
+{
+  RCTBlobManager *blobManager = [[self bridge] moduleForClass:[RCTBlobManager class]];
```

Comment:
How that could work in practice? I mean not sure how category could be used to export method. If possible - that's great and quite reasonable as long as we assume Blob is opt-in module.